docs/features/permissions.md

```diff
@@ -22,6 +22,7 @@ metadata:
       dependencies:
         - service: service-a
         - service: service-b
+          handleInternalRedirect: true
```

Comment:
If we need more such settings, I believe it would be a good moment to add a structure for it, e.g. a nested "settings" field?

Reply:
This would improve the code that follows with the injected boolean field, which would become a nice properties struct.

Reply:
It might be good change I thought about keeping same format for Incoming/Outgoing https://github.com/allegro/envoy-control/blob/733636a8ee1adf03d12ba68ee96fc21c0accabba/envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt#L109. We can discuss how we want to approach this.